core/src/scene/sceneLoader.cpp

```diff
@@ -544,7 +544,7 @@ std::shared_ptr<Texture> SceneLoader::fetchTexture(const std::string& name, cons
                 std::lock_guard<std::mutex> lock(m_textureMutex);
 				auto texture = scene->getTexture(name);
                 if (texture) {
-                    if (!texture->loadImageFromMemory(ptr, dataSize, false)) {
+                    if (!texture->loadImageFromMemory(ptr, dataSize, true)) {
```

Comment:
We can remove this param and set it to `true` by default within texture loading. It's a global state within the `stb` lib, and it can be called in worker threads that changing this option while loading another texture. Or make it thread safe, but first option sounds better.